src/collections/_documentation/platforms/javascript/angular.md

```diff
@@ -47,6 +105,48 @@ export class SentryErrorHandler implements ErrorHandler {
 export class AppModule {}
 ```
 
+When using your own `ErrorHandler`, make sure that whenever you use `HttpInterceptor` alongside it,
+it doesn't modify the error captured originally.
+The same goes for writing your own API Services with built-in `http` methods.
+
+For example, the service below make it impossible for the SDK to extract correct value.
+The reason being that it's not passed down the error pipeline in it's entirety, but returning `error` property of the original instead.
+
+```js
+export class ApiService {
+  constructor(http) {
+    this.http = http;
+  }
+
+  formatErrors(e) {
+    console.log('Captured service error');
+    return throwError(e.error);
+  }
+
+  get(path) {
+    return this.http
+      .get(path)
+      .pipe(catchError(this.formatErrors));
+  }
+}
+```
+
+Instead, make sure that you always rethrow or pass-down directly original error, eg.:
```

Comment:
```suggestion
Instead, make sure that you always rethrow or pass-down directly the original error. For example:
```

Reply:
@MimiDumpling - LOL, we edited the same sentence. Incorporated your edits into my suggestion so GH doesn't get confused, and am going to resolve this conversation.